tools/interop-node/config/config.go

```diff
@@ -116,8 +111,8 @@ func (sc *SettlusConfig) Validate() error {
 type FeederConfig struct {
 	Topics           string `yaml:"topics"`
 	Address          string `yaml:"address"`
-	SignerMode       string `yaml:"signer_mode"`
```

Comment:
We shouldn't remove these fields since it will make it more difficult to maintain as signing methods are added.